palaeovertebrata.csl

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <info>
+    <title>Palaeovertebrata</title>
+    <id>http://www.zotero.org/styles/palaeovertebrata</id>
+    <link href="http://www.zotero.org/styles/palaeovertebrata" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-harvard" rel="template"/>
+    <link href="http://palaeovertebrata.com/instructionsToAuthors" rel="documentation"/>
+    <category citation-format="author-date"/>
+	 <author>
+      <name>Renaud Lebrun</name>
+      <email>renaud.lebrun@umontpellier.fr</email>
+      <uri>http://www.mendeley.com/profiles/renaud-lebrun/</uri>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+	<issn>0031-0247</issn>
+     <eissn>2274-0333</eissn>
```

Comment:
Running this style through http://formatter.citationstyles.org/ should again correct the indentation here.